osmosis-1/osmosis.zone_assets.json

```diff
@@ -6378,6 +6378,41 @@
         "liquid_staking"
       ],
       "_comment": "MilkyWay $MILK"
+    },
+    {
+      "chain_name": "planq",
+      "base_denom": "erc20/0xF630B96760B91f6FCbE8b3C24f2786AcA3D03648",
+      "path": "transfer/channel-492/erc20/0xF630B96760B91f6FCbE8b3C24f2786AcA3D03648",
+      "osmosis_verified": false,
+      "_comment": "Anubis $ANUBI"
+    },
+    {
+      "chain_name": "planq",
+      "base_denom": "erc20/0x8F4d86ABa1b47832C46f11e0fFD0F7aDa4498345",
+      "path": "transfer/channel-492/erc20/0x8F4d86ABa1b47832C46f11e0fFD0F7aDa4498345",
+      "osmosis_verified": false,
+      "_comment": "Astonic $ATC"
+    },
+    {
+      "chain_name": "planq",
+      "base_denom": "erc20/0xA2871B267a7d888F830251F6B4D9d3DFf184995a",
+      "path": "transfer/channel-492/erc20/0xA2871B267a7d888F830251F6B4D9d3DFf184995a",
+      "osmosis_verified": false,
+      "_comment": "Astonic USD $aUSD"
```

Comment:
At the Chain Registry, could you please define the `traces` (relationship) to the currencies USD, EUR, BRL? They are defined under forex as USD, etc.

Reply:
Created a PR for it